src/components/Input/Input.scss

```
@@ -0,0 +1,121 @@
+@import '../../utils/index';
+
+$default-size: 24px;
+$big-size: 32px;
+$min-width: 40px;
+$border-width: 1px;
+$border-style: solid;
+$border-radius: 2px;
+
+.moonstone-input {
+    position: relative;
+
+    display: inline-flex;
+    align-items: center;
+    width: 100%;
+    min-width: $min-width;
+    height: $default-size;
+
+    color: var(--color-gray_dark);
+    line-height: 23px;
+
+    cursor: text;
+
+    input {
+        @extend %variant-caption;
+
+        display: block;
+        box-sizing: content-box;
+        width: 100%; // Fix IE 11 width issue
+        // Make the flex item shrink with Firefox
+        min-width: 0;
+        height: $default-size;
+        margin: 0; // Reset for Safari
+        padding: 0 var(--spacing-small);
+
+        border: $border-width $border-style var(--color-gray60);
+        border-radius: $border-radius;
+
+        -webkit-tap-highlight-color: transparent;
+
+        // Reset Firefox invalid required input style
+        &:invalid {
+            box-shadow: none;
+        }
+
+        &::placeholder {
+            font-style: italic;
+        }
+
+        &:hover,
+        &:active {
+            border-color: var(--color-gray);
+        }
+
+        &:focus {
+            border-color: var(--color-accent);
+            outline: none;
+        }
+
+        &:read-only {
+            border: transparent;
+        }
+
+        &:disabled {
+            //color: var(--color-gray);
+
+            border: transparent;
```

To keep the same height:
```suggestion
            border-color: transparent;
```

---

Updated